docs/user/release_notes.rst

```diff
@@ -50,6 +51,7 @@ The following developers contributed to Bodhi 3.3.0:
 * Till Maas
 * Ken Dreyer
 * Randy Barlow
+* Clément Verna
```

Comment:
The 3.3.0 release is already out the door and into production - can you move the two lines you changed here into a new ```develop``` heading at the top?